app/Decsys/Models/Webhooks/PayloadModel.cs

```diff
@@ -19,3 +19,32 @@ public PayloadModel(int surveyId, int instanceId, string participantId, BaseEven
     public PageResponseSummary? PageResponseSummary { get; set; } 
```

Comment:
This is an event specific property - not all events will want to provide a response summary...

Reply:
I think we should jsut change it to be a generic payload object. That way when you trigger an event, it can add whatever event specific payload it likes. This is similar to the payload object on the event object in a GitHub webhook payload.
```suggestion
    public object? Payload { get; set; } 
```